api/src/services/files.ts

```diff
@@ -60,8 +63,8 @@ export class FilesService extends ItemsService {
 					.first()) ?? null;
 		}
 
-		if (existingFile?.version || existingFile?.version === 0 || existingFile?.version === null) {
-			existingFile.version = (existingFile.version || 0) + 1;
+		if (existingFile?.version || existingFile?.version === 0) {
```

Comment:
With the changes to the schema and expected type of `version` can this not simply a be `if(existingFile) {`? Since we know that if we have an existing file we will always have a version as well?

Reply:
You be 100% correct.